packages/kinvey-nativescript-sdk/lib/before-checkForChanges.js

```diff
@@ -46,6 +46,13 @@ module.exports = function (hookArgs) {
           if (currentContent !== micInfoPlistFileContent) {
             fs.writeFileSync(destinationInfoPlistFile, micInfoPlistFileContent);
           }
+        } else {
+          const defaultInfoPlistFile = path.join(__dirname, '..', 'platforms', 'ios', 'Info.default.plist');
```

Comment:
Not sure if capitalization matters, but the file at the end of the path is `Info.default.plist`, but the actual filename is `info.default.plist`. May be worth changing it to be the same for both.

Reply:
Good catch. This has been fixed.